app/helpers/save_this_page_helper.rb

```diff
@@ -0,0 +1,32 @@
+module SaveThisPageHelper
+  attr_reader :page_is_saved, :signed_in, :page_path
+
+  PAGE_IS_SAVED_LINK_HREF = {
+    true => "/account/delete-a-page?page_path=",
```

Comment:
Ok I think those are set now, see here: https://github.com/alphagov/special-route-publisher/pull/165/commits/6342e5120be2ac1e502160af28520bef486ca55c

I'll follow this up with a suggestion!

Reply:
```suggestion
    true => "/account/saved-pages/add?page_path=",
```